examples/quick_template_score_calc.ipynb

```diff
@@ -528,7 +520,7 @@
    "source": [
     "temperature_score = TemperatureScore(\n",
     "    time_frames = [ETimeFrames.LONG],\n",
-    "    scopes=[EScope.S1S2],\n",
+    "    scopes=[EScope.S1S2, EScope.S3],\n",
     "    aggregation_method=PortfolioAggregationMethod.WATS # Options for the aggregation method are WATS, TETS, AOTS, MOTS, EOTS, ECOTS, and ROTS.\n",
```

Comment:
From slack chat with Michael:

these two print the same data, whereas I would expect them to print different data since the scope/temperature scores are different:

~~~~
enhanced_portfolio[(enhanced_portfolio.scope==EScope.S1S2)&(enhanced_portfolio.temperature_score>Q_(3, 'delta_degC'))]

enhanced_portfolio[(enhanced_portfolio.scope==EScope.S3)&(enhanced_portfolio.temperature_score>Q_(3, 'delta_degC'))]
~~~~

Reply:
This finding is caused by the other finding in Jupiter notebook for S3, so I will resolve this duplicate entry